app/javascript/stylesheets/post.scss

```diff
@@ -0,0 +1,48 @@
+
+.post-container{
```

Comment:
As we are using bootstrap, it would be wiser to use the built in grid system instead of the `@media` rule.
Also you there is a really nice bootstrap component that has almost everything in this scss file: [card](https://getbootstrap.com/docs/5.0/components/card/)

I am not really good in front-end work, so we should consult @triszt4n  on this matter.

Reply:
I think it is simpler to use a media query here.
I think the card component is not exactly for this application.

@triszt4n what do yout think?